docs/documentation/actions/micrometers.md

```diff
@@ -0,0 +1,285 @@
+# Counter
+
+Use this action to report
+a [count](https://micrometer.io/docs/concepts#_counters:~:text=9.-,Counters,-Counters%20report%20a){:target=_blank}
+metric.
+
+=== "Inputs"
+
+    | Required | Name         | Type                             |  Description                                             |
+    |:--------:|:-------------|:---------------------------------|:---------------------------------------------------------|
+    |    *     | `name`       | String                           | Counter name. It's required if counter input is null.    |
+    |    *     | `counter`    | [Counter](https://javadoc.io/doc/io.micrometer/micrometer-core/latest/io/micrometer/core/instrument/Counter.html){:target=_blank}             | Counter instance. It's required if name input is null. |
+    |          | `description`| String                           | Counter  description                                 |
+    |          | `unit`       | Map <String, String\>            | Count [unit](https://github.com/micrometer-metrics/micrometer/blob/main/micrometer-core/src/main/java/io/micrometer/core/instrument/binder/BaseUnits.java){:target=_blank}                                                 |
+    |          | `tags`       | List<String\>                    | key,value list representing tags. A tag is a Key/value pair representing a dimension of a meter used to classify and drill into measurements.                                  |
+    |          | `registry`   | [MeterRegistry](https://javadoc.io/doc/io.micrometer/micrometer-core/latest/io/micrometer/core/instrument/MeterRegistry.html){:target=_blank}             | Creates and manages your application's set of meters  |
```

Comment:
question for us : why not always globalRegistry ? What are use cases where we want to use another one ?